src/NHSD.BuyingCatalogue.Solutions.API/Controllers/AdditionalServiceController.cs

```diff
@@ -10,39 +15,44 @@ namespace NHSD.BuyingCatalogue.Solutions.API.Controllers
     [Produces(MediaTypeNames.Application.Json)]
     public sealed class AdditionalServiceController : ControllerBase
     {
+        private readonly IMediator _mediator;
+
+        public AdditionalServiceController(IMediator mediator)
+        {
+            _mediator = mediator ?? throw new ArgumentNullException(nameof(mediator)); ;
```

Comment:
double semicolon at the end of this line

Reply:
👍